doc/admin/deployment.md

```diff
@@ -87,282 +125,183 @@ in order to access NVMe SSDs through the SPDK framework.
 
-This step requires a reboot to enable DCPM resource allocation
-changes to be read by BIOS.
+The DAOS security framework relies on certificates to authenticate
+administrators. The security infrastructure is currently under development and
+will be delivered in DAOS v1.0. Initial support for certificates has been added
+to DAOS and can be disabled either via the command line or in the DAOS server
+configuration file. Currently, the easiest way to disable certificate support
+is to pass the -i flag to daos_server.
 
-DCPM preparation can be performed from the management tool
-`dmg storage prepare --scm-only` or using the Control Server directly
-`sudo daos_server storage prepare --scm-only`.
+### Server Startup
 
-After running the command a reboot will be required, then the Control
-Servers will then need to be started again and the command run for a
-second time to expose the namespace device to be used by DAOS.
-([more details](#scm-preparation))
+One instance of the `daos_server` process is to be started per
+storage node. The server can be started either individually (e.g. independently
+on each storage node via systemd) or collectively (e.g. pdsh, mpirun or as a
+Kubernetes Pod).
 
-Example usage:
-- `dmg -l wolf-[118-121,130-133] -i storage prepare --scm-only`
-after running, the user should be prompted for a reboot.
-- `clush -w wolf-[118-121,130-133] reboot`
-- `clush -w wolf-[118-121,130-133] daos_server start -o utils/config/examples/daos_server_sockets.yml`
-- `dmg -l wolf-[118-121,130-133] -i storage prepare --scm-only`
-after running, `/dev/pmemX` devices should be available on each of the hosts.
+#### Parallel Launcher
 
-### Step 4. Scan Storage to check device details and state
+Practically any parallel launcher can be used to start the DAOS server
+collectively on a set of storage nodes. pdsh, clush and orterun are most
+commonly used.
 
-- `dmg -l wolf-[118-121,130-133] -i storage scan [--verbose]`
-will display details of NVMe and DCPM devices on the specified hosts that can be used with DAOS.
-[more details](#storage-detection--selection)
+```bash
+$ clush -w <server_list> -o "-t -t" daos_server start -o <config_file>`
+```
+will launch `daos_server` on the specified hosts connecting to the `port`
+parameter value specified in the server config file.
+If the number of storage node exceed the default fanout value, then "-f"
+followed by the number of storage nodes should be used.
 
-Note that other storage query commands are also available,
-`dmg storage --help` for listings.
+Similarly, pdsh can be used:
+```bash
+$ pdsh -w <server_list> daos_server start -o <config_file>`
+```
 
-### Step 5. Add device identifiers to server config file
+As for orterun, the list of storage nodes can be specified on the command line
+via the -H option. To start the DAOS server, run:
+```bash
+$ orterun --map-by node --mca btl tcp,self --mca oob tcp -np <num_servers>
+-H <server_list> --enable-recovery daos_server start -o <config_file>
+```
```

Comment:
Most recently, during some work to get ofi+verbs;ofi_rxm tests running, Alex had me also specify "--mca pml ob1" with orterun.  Can you check with Alex to see if that is a change that should be in there moving forward?

Reply:
will check.